src/ex1/script.js

```diff
@@ -0,0 +1,123 @@
+
+function addTask(){
+    const task = document.getElementById('newTask');
+    // check if entered empty task
+    if (task.value === ''){
+        alert("You forgot to type the task!");
+        return;
+    }
+    if (document.getElementById(task.value) !== null){
```

Comment:
Having the id of the DOM element as the id of the task AND the value presents a limitation
1. A user should be able to enter the same task multiple times
2. I can't enter tasks with the same value as existing DOM elements
 
A better solution would be to save the tasks in a list in memory with unique ids 

![image](https://user-images.githubusercontent.com/36815567/169354332-22cb600d-a752-480f-aef4-e4c6bfb032c0.png)

Reply:
yeah I did that on purpose... doesn't make sense- fixed it